client/scss/components/_listing.scss

```diff
@@ -543,14 +543,6 @@ table.listing {
           text-align: center;
           height: 180px;
 
-          &:before {
-            content: '';
-            display: inline-block;
-            height: 100%;
-            vertical-align: middle;
-            margin-inline-end: -0.25em;
-          }
```

Comment:
This isn’t an icon, but those styles do nothing useful as far as I can tell. It’d be safer to leave them in, but I really don’t see what this does.

Reply:
This allows the images on the image listing to be vertically centered on each row. I know we can achieve this with more modern means e.g. grid, but I'd suggest leaving this out of the PR for now:

<details>
<summary>Screenshots</summary>



## Before

<img width="1119" alt="image" src="https://github.com/wagtail/wagtail/assets/6379424/d7b69117-fa54-4a61-af70-85bdac3286d2">


## After

<img width="1119" alt="image" src="https://github.com/wagtail/wagtail/assets/6379424/72f57232-3d82-4360-8a6c-50c197025b65">


</details>

Reply:
Ah! Good spotting, thank you @laymonage.